### PR TITLE
Pipeline support

### PR DESCRIPTION
**Pull request by gaffa**
Support for jenkins pipelines

### CODE REVIEW

**Comment by gaffa** [comment]
@ulich @CanardSauvage : do you like to review?

File: dash-core/src/main/java/de/axelspringer/ideas/tools/dash/business/jenkins/domain/JenkinsJobInfo.java [hunk 1]
```diff
@@ -1,32 +1,54 @@
...
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 import org.apache.commons.lang3.builder.ToStringBuilder;
 
 public class JenkinsJobInfo {
 
-    private LastBuild lastCompletedBuild;
+    private final static String PIPELINE_CLASS = "org.jenkinsci.plugins.workflow.job.WorkflowJob";
 
-    private LastBuild lastBuild;
+    private String _class;
```
Comment (ulich) [inline dash-core/src/main/java/de/axelspringer/ideas/tools/dash/business/jenkins/domain/JenkinsJobInfo.java]: using @JsonProperty here might be cleaner, don't you think?

**Comment by ulich** [comment]
rest looks good, maybe add a test for the pipeline stuff somewhen :)

**Comment by gaffa** [comment]
i incorporated all your feedback and wrote some tests. can you please review again?

**Comment by ulich** [comment]
looks good, you can merge

**Comment by CanardSauvage** [comment]
Totally crashed the pcp monitor :(

**Comment by ulich** [comment]
please provide further information and we fix it. compilation or runtime error?

**Comment by CanardSauvage** [comment]
I already fixed it ;) Maybe we should think about versioning?

**Comment by ulich** [comment]
yeah. we did something like this here: https://github.com/upday/android-devicemanager/blob/master/travis-build.sh

every time travis builds the master branch, it sets the version to the current build number and deploys it. Its not maven-style but rather pragmatic :)

**Comment by CanardSauvage** [comment]
Totally did the same but with jenkins ;) Thanks a lot Timo!

**Pull request merged**